lib/matplotlib/backends/backend_agg.py

```diff
@@ -460,9 +459,7 @@ def _print_pil(self, filename_or_obj, fmt, pil_kwargs, metadata=None):
             filename_or_obj, self.buffer_rgba(), format=fmt, origin="upper",
             dpi=self.figure.dpi, metadata=metadata, pil_kwargs=pil_kwargs)
 
-    @_api.delete_parameter("3.5", "args")
-    def print_png(self, filename_or_obj, *args,
-                  metadata=None, pil_kwargs=None):
+    def print_png(self, filename_or_obj, metadata=None, pil_kwargs=None):
```

Comment:
Do we wish for metadata and pil_kwargs to remain keyword only?

Similar question for all the other places where `*args` is removed with named kwargs

Reply:
oh, good catch!

Reply:
Fixed!